viz.yaml

```diff
@@ -109,28 +109,28 @@ process:
     location: cache/process/lastUpdated.yaml
     scripts: scripts/process/lastUpdated.R
     depends:
-      data: aggregate_ga
+      data: fetchGA
     mimetype: text/yaml
     reader: md # need a better plain text reader
   -
     id: sessions_and_new_users
     processor: sessions_and_new_users 
     scripts: scripts/process/sessions_and_new_users.R
-    depends: aggregate_ga
+    depends: fetchGA
```

Comment:
these scripts probably depend on the name of the dependency, can't just swap out due to this.